docker-image/LICENSE

```diff
@@ -0,0 +1,176 @@
+
```

Comment:
Is this just a copy of the main `LICENSE.md` file?

Reply:
This is a copy of the main LICENSE. The root `LICENSE` is now a symlink to this file.

Reply:
Actually, I've removed the need for the symlink in 862cc4a